site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,25 +38,24 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
-[field]#displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
-A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
+[field]#identityProviderLink.displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human-readable value identifying the user in the remote identity provider.
 +
-This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.
+After the link is created, and a user successfully logs in for the first time with the IdP, this value will typically be set to an email or username.
 
+[field]#identityProviderLink.identityProviderId# [type]#[UUID]# [required]#Required#::
+The Id of the existing Identity Provider.
```

Comment:
Can we just say `The Id of the identify provider you wish to link to the user.`?